.pre-commit-config.yaml

```diff
@@ -56,3 +56,9 @@ repos:
       - id: isort
         files: \.py$
         args: ["--profile", "black", --line-length=79]
+
+  # Ensure GitHub workflows match the expected schema.
+  - repo: https://github.com/sirosen/check-jsonschema
```

Comment:
Wow - this is really cool. i've definitely pushed files with yaml syntax errors before.

Reply:
There's a few more related schemas there + the ability to host a custom one locally. Though, sometimes those schemas have bugs too.